iabtcf-gvl/src/main/java/com/iabtcf/gvl/v2/GvlV2Util.java

```diff
@@ -0,0 +1,78 @@
+package com.iabtcf.gvl.v2;
+
+/*-
+ * #%L
+ * IAB TCF Core Library
+ * %%
+ * Copyright (C) 2020 IAB Technology Laboratory, Inc
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.iabtcf.gvl.v2.dao.GvlData;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class GvlV2Util {
+    public static final String DEFAULT_BASE_GVL_URL = "https://vendorlist.consensu.org/v2/archives/vendor-list-v";
```

Comment:
GvlSyncImplCache and GvlAsyncImplCache are reference implementations of the cache. I am guessing IAB has its own way to blacklist vendors if they overuse their API.  I have made that very clear in the Javadoc of those classes. But if its making you uncomfortable, then I don't mind deleting those classes. 

The current implementation will only hit the IAB API once for a specific version forever.  Now, if the application using this library has multiple servers, then yes it will be once per server.  I am not sure such an application will use our cache, as they usually have their own caching mechanism. With the GvlCache interface, they know how to plug it in.